snuba/datasets/storages/replays.py

```diff
@@ -58,7 +66,14 @@
     storage_set_key=StorageSetKey.REPLAYS,
 )
 
-# TODO: set up deadletter queue for bad messages.
+
+def produce_policy_creator() -> DeadLetterQueuePolicy:
+    """Produce all bad messages to dead-letter topic."""
+    return ProduceInvalidMessagePolicy(
+        KafkaProducer(build_kafka_producer_configuration(Topic.DEAD_LETTER_METRICS)),
+        KafkaTopic(Topic.DEAD_LETTER_METRICS.value),
```

Comment:
You will need your own topic. That is the Topic we use for the metrics consumer.
Please add one in the topics file here https://github.com/getsentry/snuba/blob/master/snuba/utils/streams/topics.py#L28-L31

Reply:
👍